File: RentalManager.WebApp/src/references/codegen/Documents/models/Agreement.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { Client } from './Client';
import type { Equipment } from './Equipment';
import type { Payment } from './Payment';

export type Agreement = {
    client: Client;
    equipments: Array<Equipment>;
    payments: Array<Payment>;
    transport_from?: number;
    transport_to: number;
    deposit: number;
};

File: RentalManager.Global/Requests/GetAgreementDocument/Agreement.cs
// ReSharper disable UnusedAutoPropertyAccessor.Global
namespace RentalManager.Global.Requests.GetAgreementDocument;

public class Agreement
{
    public Client Client { get; init; } = null!;

    public IList<Equipment> Equipments { get; init; } = null!;

    public IList<Payment> Payments { get; init; } = null!;

    public int? TransportFrom { get; init; }

    public int TransportTo { get; init; }

    public int Deposit { get; init; }
}